index.bs

```diff
@@ -309,15 +311,21 @@ partial interface ModelBuilder {
 ### clamp ### {#api-modelbuilder-clamp}
 Clamp the input tensor element-wise within a range specified by the minimum and maximum values.
 <script type=idl>
+dictionary ClampOptions {
+  Operand minValue;
+  Operand maxValue;
+};
+
 partial interface ModelBuilder {
-  Operand clamp(Operand x, optional Operand minValue, optional Operand maxValue);
+  Operand clamp(Operand x, optional ClampOptions options = {});
```

Comment:
ditto, the sample code would be changed.

Reply:
Fixed